packages/x-date-pickers/src/internals/hooks/validation/useDateValidation.ts

```diff
@@ -16,12 +16,6 @@ export interface DateComponentValidationProps<TDate>
     YearValidationProps<TDate>,
     Required<BaseDateValidationProps<TDate>> {}
 
-export interface DateComponentDefaultizedValidationProps<TDate>
```

Comment:
This interface was 100% equivalent to `DateComponentValidationProps` above
Maybe it will be interesting to have the defaultized and non defaultized interfaces in the future but for now it was just confusing